build.sbt

```diff
@@ -343,6 +343,28 @@ lazy val iceberg = (project in file("iceberg"))
     assembly / assemblyJarName := s"${name.value}_${scalaBinaryVersion.value}-${version.value}.jar",
     assembly / logLevel := Level.Info,
     assembly / test := {},
```

Comment:
overall, I think this works, but I hate this approach. its brittle, we are eliminating on each dependency of delta-spark explicitly one at a time. And in multiple places. Tomorrow if there is another dependency added (say delta-spark depends on delta-kernel-XYZ), this will break again. 

Can you try to find any way to generalize this?